server/v2/api/grpc/server.go

```diff
@@ -108,6 +108,10 @@ func (g *GRPCServer[AppT, T]) Config() any {
 	return g.config
 }
 
+func (g *GRPCServer[AppT, T]) SetConfig(config *Config) {
```

Comment:
What's this?

Reply:
for writing  custom grpc config. With each node I created a temp server with comet & grpc in `init-testnet`, these servers will not be used actually just write config file. Then start will read config for server initial.

Reply:
instead of writing file specifically in cmd

Reply:
Right, but outside that command you should never need to overwrite the defaults that way right? Plus, this will get overwritten at init, I think. I think we need a better way to do config override. 

Personally, I think it would be nicer if this is an option (as variadic, so it's not mandatory) on the constructor to set the custom config. We need to update init to as well use `s.Config()` instead of the default config.

Opened https://github.com/cosmos/cosmos-sdk/pull/20874 for it

Reply:
updated